cheroot/test/test_cli.py

```diff
@@ -0,0 +1,37 @@
+from cheroot.cli import (
+    Application,
+    parse_wsgi_bind_addr
+)
+
+
+def test_parse_wsgi_bind_location_for_tcpip():
+    assert parse_wsgi_bind_addr('192.168.1.1:80') == ('192.168.1.1', 80)
+    assert parse_wsgi_bind_addr('[::1]:8000') == ('::1', 8000)
+
+
+def test_parse_wsgi_bind_location_for_unix_socket():
+    assert parse_wsgi_bind_addr('/tmp/cheroot.sock') == '/tmp/cheroot.sock'
+
+
+def test_parse_wsgi_bind_addr_for_abstract_unix_socket():
+    assert parse_wsgi_bind_addr('@cheroot') == '\0cheroot'
+
+
+def test_Aplication_resolve():
+    import sys
```

Comment:
Imports like this belong to the top of the module. But maybe you could even avoid it with the use of `monkeypatch` fixture.